submissions/2025-09-02-tmQM-xtb-Dataset-T=100K-low-mw-high-coordinate-mult=3-v0.0/environment.yml

```diff
@@ -0,0 +1,22 @@
+name: qca-clean
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+  - numpy
+  - jupyter
+  - pandas
+  - h5py
+  - periodictable
+  - qcportal>=0.61
+  - qcfractal>=0.61
+  - qcfractalcompute>=0.59
+  - rdkit>=2025.3.3
+  - openbabel
+  - deepdiff
+  - py3Dmol
+  - scipy
+  - networkx
+  - pip:
+      - git+https://github.com/openforcefield/tmos.git
+      - git+https://github.com/MDAnalysis/mdanalysis.git@develop#subdirectory=package # delete after 2.10 is released
```

Comment:
FYI, you can replace `develop` with a specific hash commit if you want to be super specific.

Reply:
Thanks for letting me know! It just needs to be newer than the current branch so I don't expect an issue